src/posts/2020-10-28-dyslexia-fonts.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: Dyslexia fonts
+description: A quick overview of dyslexia fonts
+category: Assistive technology
+author: Hamsa Harcourt
+further_reading:
+  - url: https://accessibility.huit.harvard.edu/design-readability
+    title: Design for readability
+    source: Havard University
+  - url: https://eyeondesign.aiga.org/sad-but-true-comic-sans-might-just-be-the-best-font-for-dyslexics/
+    title: Comic Sans Might Just be the Best Font for People with Dyslexia
+    source: AIGA Eye on Design
+date: 2020-10-28
+eleventyNavigation:
+  key: {{ category }}
+  title: {{ title }}
+tags:
+  - assistivetechnology
+---
+
+<blockquote>
+  <p>
+    Dyslexia is a slight disorder of the brain that causes difficulty in reading and spelling, for example, but does not affect intelligence.
+  </p>
+		<footer>
+		<cite><a href="https://www.oxfordlearnersdictionaries.com/definition/english/dyslexia?q=dyslexia">Dyslexia - Oxford Advanced Learner's Dictionary</a></cite>
+	</footer>
+</blockquote>
+
+Dyslexia affects about 1 in 5 people worldwide. Its symptoms range in severity from mild to extreme. It transposes letters, numbers, punctuation, and other characters, which affects your ability to read. This makes selecting Dyslexic-friendly fonts an important thing to consider.
```

Comment:
Does it make Dyslexic-friendly fonts important to consider? Later on you reveal that they're not effective. It might be better to say something like, "This means that making Dyslexic-friendly fonts and typography choices important to consider.", as those are the meaningful changes a designer can do.

Reply:
@HarcourtHamsa Tagging you on unresolved feedback.